TransformatieRDF.md

```diff
@@ -1607,61 +1605,89 @@ WHERE {
       ?enumeratiewaarde mim:code ?notation
     }
   }
-  BIND (t:classeuri(?notation) as ?class)
+  BIND (t:uri(?notation) as ?class)
 }
 </pre>
+### transformatie: bevat modelement
 
-### transformatie: bevat modelelement
-
-Een `mim:bevatModelelement` wordt vertaald naar een `rdfs:isDefinedBy` voor zover modelelementen vertaald worden naar een owl:Class, owl:DatatypeProperty of owl:ObjectProperty. Merk op dat de vertaling in de omgekeerde richting is. Als bijvoorbeeld een Domein X modelelement Objecttype Y bevat, dan zal de relatie rdfs:isDefinedBy gaan lopen vanaf de klasse Y naar de Ontology X. In geval er sprake is van een owl:Ontology, dan wordt vertaald naar `owl:imports`.
+Een `mim:bevatModelelement` wordt vertaald naar een `mim:bevatModelelement` In geval er sprake is van een owl:Ontology, dan wordt vertaald naar `owl:imports`.
```

Comment:
Klopt dit wel? De originele beschrijving geeft juiste aan dat de boel wordt omgedraaid. Dat is toch wat wilt?

Reply:
mim:isGedefinieerdIn is toegevoegd. Dit is de tegenhanger van rdfs:isDefinedBy. Nu is het niet meer nodig om mim:bevatModelement te gebruiken (wat eigenlijk ook een net andere betekenis heeft). Met de tekst was niks mis want dit staat nu bij ### transformatie: is gedefinieerd in.